src/main/mw.c

```diff
@@ -523,8 +523,13 @@ void taskMainPidLoop(void)
     cycleTime = getTaskDeltaTime(TASK_SELF);
     dT = (float)cycleTime * 0.000001f;
 
-    imuUpdateAccelerometer();
-    imuUpdateGyroAndAttitude();
+    /*
+     * Acc update is run in sync with PID loop only when RACE mode is not enabled
+     */
+    if (!feature(FEATURE_RACE) && sensors(SENSOR_ACC)) {
```

Comment:
Probably, for now I will keep it as it is. I really would like to determine how low I can go down with ACC sampling before aliasing effects will appear